add_item.php

```diff
@@ -0,0 +1,54 @@
+<?php
+require_once('functions.php');
+
+if (isset($_POST['organism']) &&
+    isset($_POST['incubation_usual']) &&
+    isset($_POST['incubation_range']) &&
+    isset($_POST['symptoms']) &&
+    isset($_POST['severity']) &&
+    isset($_POST['avg_annual_incidence']) &&
+    isset($_FILES['img']) &&
+    $_FILES['img']['name'] != '') {
+    $imgFileName = moveUploadedImgToFolderAndGrabName();
+    $db = establishDisease_dbConnection();
+    addNewDiseaseToDB($db, $imgFileName);
+}
+
+
+?>
+<!DOCTYPE html>
+<html>
```

Comment:
html tag missing lang attribute

Reply:
I think this is the second time for the same thing, probably need to put your boilerplate into an html validator like https://validator.w3.org/ to make sure it's valid